### PR TITLE
Docker deployment

### PR DESCRIPTION
**Pull request by Minosity-VR**
Add dockerfiles for frontend and marsha-mimicking backend.

Docker-compose file to deploy everything, plus a Certbot container to generate SSL certificates.

Add a custom script to ease the first launch, as Nginx cannot start without certificates and Certbot needs Nginx container to be up and running to generate the first certificates.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Apple folder misc info
-.DS_Store
\ No newline at end of file
+.DS_Store
```
Comment (mkgharbi) [inline .gitignore:2]: No difference ? Right ?

File: staging/fake-marsha-backend/app.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ app.use(cors());
 
 // A sample route
 app.get('/', (req, res) => res.send('Hello World!'))
```
Comment (mkgharbi) [inline staging/fake-marsha-backend/app.js]: no more useful line normally. Right ?

File: .gitignore (hunk 1 above)
Comment (Minosity-VR) [reply .gitignore:2]: Just added a blank newline at the end of file, to remove the ugly '⛔' at the end of the line

File: staging/fake-marsha-backend/app.js (hunk 2 above)
Comment (Minosity-VR) [reply staging/fake-marsha-backend/app.js]: No, effectively, fixing it soon

**Review by mkgharbi — APPROVED** [review]

**Pull request merged**